### PR TITLE
Implement getTransactionObjects command

### LINKED ISSUE #241: Missing get transaction objects command
The JS library implements `getTransactionObjects` command, that is essentially the same as `findTransactionObjects`, but awaits transaction hashes as inputs.

To make PyOTA more consistent with other libraries, implement the command.

### PR DESCRIPTION
**Pull request by lzpap**
New command to harmonize PyOTA with JS lib.
Similar to findTransactionObjects, but input
is a list of transaction hashes.

resolves #241

### CODE REVIEW

File: iota/commands/extended/get_transaction_objects.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+__all__ = [
+    'GetTransactionObjectsCommand',
+]
+
+
+class GetTransactionObjectsCommand(FilterCommand):
+    """
+    Executes `GetTransactionObjectsCommand` command.
+
+    See :py:meth:`iota.api.StrictIota.get_transaction_objects`.
```
Comment (todofixthis) [inline iota/commands/extended/get_transaction_objects.py]:
```suggestion
    See :py:meth:`iota.api.Iota.get_transaction_objects`.
```

(`StrictIota` contains only commands exposed by the IRI node; original inspiration taken from [redis](https://github.com/andymccurdy/redis-py/blob/2.10.6/redis/client.py#L347-L356))

File: iota/commands/extended/get_transaction_objects.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    def get_request_filter(self):
+        return GetTransactionObjectsRequestFilter()
+
+    def get_response_filter(self):
+        pass
+
+    def _execute(self, request):
+        hashes = request\
+            .get('hashes') # type: Optional[Iterable[TransactionHash]]
```
Comment (todofixthis) [inline iota/commands/extended/get_transaction_objects.py]:
```suggestion
            .get('hashes') # type: Iterable[TransactionHash]
```

(the `f.Required` filter in the request filter chain ensures that this value cannot be `None`)

File: test/commands/extended/get_transaction_objects_test.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+from unittest import TestCase
+
+import mock
+
+from iota import Iota, MockAdapter, Transaction
+from iota.commands.extended import GetTransactionObjectsCommand
+
+
+class GetTransactionObjectsCommandTestCase(TestCase):
```
Comment (todofixthis) [inline test/commands/extended/get_transaction_objects_test.py]: ❔ Should we also include tests for `GetTransactionObjectsRequestFilter`?

Comment (todofixthis) [reply test/commands/extended/get_transaction_objects_test.py]: See #243

File: iota/commands/extended/get_transaction_objects.py (hunk 1 above)
Comment (lzpap) [reply iota/commands/extended/get_transaction_objects.py]: Thanks for noticing!

File: iota/commands/extended/get_transaction_objects.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        return {
+            'transactions': transactions,
+        }
+
+class GetTransactionObjectsRequestFilter(RequestFilter):
+    def __init__(self):
+        super(GetTransactionObjectsRequestFilter, self).__init__({
+            'hashes':
+                StringifiedTrytesArray(TransactionHash) | f.Required
+                ),
```
Comment (todofixthis) [inline iota/commands/extended/get_transaction_objects.py]:
```suggestion
```

Comment (lzpap) [reply iota/commands/extended/get_transaction_objects.py]: whoops

**Pull request merged**